migrations/3_deploy_nori.js

```diff
@@ -7,7 +7,7 @@ const ContractRegistryV0_1_0 = artifacts.require('ContractRegistryV0_1_0');
 const RootRegistryV0_1_0 = artifacts.require('RootRegistryV0_1_0');
 const NoriV0_1_0 = artifacts.require('NoriV0_1_0');
 const ParticipantRegistryV0 = artifacts.require('ParticipantRegistryV0');
-const CRCV0_1_0 = artifacts.require('CRCV0_1_0');
+const CRCV0_2_0 = artifacts.require('CRCV0_2_0');
```

Comment:
this is where it would be nice to have a wrapper like:
```
const CRC = artifacts.getLatest('CRC');
```
which would just give you the latest version.